x-pack/heartbeat/monitors/browser/source/zipurl_test.go

```diff
@@ -44,7 +44,7 @@ func TestSimpleCases(t *testing.T) {
 			mapstr.M{
 				"folder":           "/",
 				"retries":          3,
-				"target_directory": "/tmp/synthetics/blah",
+				"target_directory": filepath.Join(os.TempDir(), "synthetics", "blah"),
```

Comment:
This makes running tests natively on windows possible, and is more correct anyway, we should always use `filepath` to join dirs where possible.